lua/entities/gmod_wire_cd_lock.lua

```diff
@@ -20,7 +20,7 @@ function ENT:Initialize()
 	self.DisableLinking = 0
 
 	self.Inputs = Wire_CreateInputs(self, { "Disable" })
-	self.Outputs = Wire_CreateOutputs(self, { "Locked" })
+	self.Outputs = Wire_CreateOutputs(self, { "Locked", "DiskEntity [ENTITY]" })
```

Comment:
Can you change this to use `WireLib.CreateSpecialOutputs`? The old style (where WireLib has to parse text) should be got rid of wherever possible.

Reply:
yes sure, i just didn't know about that) going to do that quickly